utils/useragent/useragent.go

```diff
@@ -0,0 +1,30 @@
+package useragent
+
+import (
+	"github.com/varstr/uaparser"
```

Comment:
I noticed this parser isn't actively maintained - is there an advantage over something like https://github.com/mssola/user_agent ?

Reply:
oh, no i can swap that too

Reply:
fixed in: 49fdac0